spec/controllers/lambda_callback/document_proof_result_controller_spec.rb

```diff
@@ -20,36 +19,25 @@
     context 'with valid API token' do
       before do
         request.headers['X-API-AUTH-TOKEN'] = AppConfig.env.document_proof_result_lambda_token
-
-        document_capture_session.store_proofing_pii_from_doc(pii)
```

Comment:
this part makes zero sense in context, we don't have the PII until this callback is called, so it was very important to remove it